airbyte-cron/src/main/resources/application.yml

```diff
@@ -0,0 +1,215 @@
+micronaut:
+  application:
+    name: airbyte-workers
+  security:
+    intercept-url-map:
+      - pattern: /**
+        httpMethod: GET
+        access:
+          - isAnonymous()
+  server:
+    port: 9000
+
+airbyte:
```

Comment:
I am a bit confused about this part. It looks like we are starting to duplicate configs and a lot of the keys here feel irrelevant to the `airbyte-cron` itself.

Reply:
@gosusnp Agreed.  We shouldn't need the full config from the `airbyte-workers` branch.  We should only need the config bits needed to configure the Temporal client.  I would start by commenting them all out and then restoring the ones that are actually used (Micronaut will complain at runtime that it can't find the property).

Reply:
@jdpgrailsdev @gosusnp I answered it here: https://github.com/airbytehq/airbyte/pull/16414#discussion_r966364804 I will bring it to the sit and work.